src/core/gltf2importer/animationparser.cpp

```diff
@@ -361,8 +383,8 @@ Qt3DAnimation::QChannel AnimationParser::animationChannelDataFromBuffer(const QS
 
 bool AnimationParser::checkSamplerJSON(const QJsonObject &samplerObject) const
 {
-    const QJsonValue &inputValue = samplerObject[KEY_INPUT];
-    const QJsonValue &outputValue = samplerObject[KEY_OUTPUT];
+    const QJsonValue inputValue = samplerObject.value(KEY_INPUT);
```

Comment:
probably for future patches, we should avoid modifiying this kind of stuff. It add noise to the feature we are trying to review

Reply:
well it's done as a dedicated commit https://github.com/KDAB/kuesa/pull/79/commits/ae8c202138e5412c6524dec2267c881bd44fc494
it's just that if you review with the all files changes, you don't see that.